### PR TITLE
docs: Improve quick start

### PR DESCRIPTION
**Pull request by npentrel**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Docs updates for the Quick Start page.

- **Why was this change needed?** (You can also link to an open issue here)
Improve Readability.

- **Other information**:

### CODE REVIEW

File: docs/docs/overview/quick-start.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 2
 
 # Quick Start
 
-To create your free managed, or docker based Novu environment use our CLI tool:
+To create your free managed or docker-based Novu environment use our CLI tool:
```
Comment (p-fernandez) [inline docs/docs/overview/quick-start.md:7]: Please roll back as the existing copy is ok.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Also please solve the merge conflict so we can run the integration checks. Thank you.

**Comment by npentrel** [comment]
All good to go now!

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**